setup.py

```diff
@@ -8,9 +8,9 @@
 
 extras_require = {
     'test': [
-        "pytest==3.3.2",
-        "tox>=2.9.1,<3",
-        "hypothesis==3.56.5",
+        "pytest==5.3.0",
+        "tox",
```

Comment:
Can you pin this one within the minor version range of whatever is the latest released version.

Reply:
Done! `tox>=3.14.0,<3.15.0`